### ISSUE TITLE
Problem using net

### ISSUE THREAD

**Issue by professormahi** [vladikoff/chromeos-apk#153]
I'm trying to install line and clash of clans on my chrome but it seams that the application dosen't have connection to internet and can't access internet please help

**Comment by zhangqi-net** [comment]
Yes I have the same problem. After install some apps they won't connect to internet.
This issue might possible caused by certain apps.
